x-pack/plugins/threat_intelligence/public/modules/indicators/indicators_page.tsx

```diff
@@ -16,9 +16,16 @@ import { FiltersGlobal } from '../../containers/filters_global';
 import QueryBar from '../query_bar/components/query_bar';
 import { useSourcererDataView } from './hooks/use_sourcerer_data_view';
 import { FieldTypesProvider } from '../../containers/field_types_provider';
+import { InspectorProvider } from '../../containers/inspector';
 import { useColumnSettings } from './components/indicators_table/hooks/use_column_settings';
 
-export const IndicatorsPage: VFC = () => {
+const IndicatorsPageProviders: FC = ({ children }) => (
+  <FieldTypesProvider>
+    <InspectorProvider>{children}</InspectorProvider>
+  </FieldTypesProvider>
+);
```

Comment:
just for my own curiosity, any reason you separated this (and `IndicatorsPage` below line 85)?

Reply:
I just wanted to clean this up a bit.. eventually, it could go to a separate file I think.